src/ts_catalog/continuous_agg.c

```diff
@@ -1493,7 +1503,7 @@ ts_compute_inscribed_bucketed_refresh_window_variable(int64 *start, int64 *end,
  */
 void
 ts_compute_circumscribed_bucketed_refresh_window_variable(int64 *start, int64 *end,
-														  const ContinuousAggsBucketFunction *bf)
+														  ContinuousAggsBucketFunction *bf)
```

Comment:
Looks like the usage of `bf` is still const, why is it not const anymore?

Reply:
I would work around this one function to keep the const signature for many others. Anyway, const in C is not so useful.

By the way, the namestrcmp is logically const and should probably be patched upstream, do you want to send them a patch? They're merging them surprisingly fast lately :)

Reply:
Thanks. Good idea. Would be my first upstream contribution. :)

Reply:
Reverted to const